wordlist.txt

```diff
@@ -54,3 +54,14 @@ routable
 milliCPU
 overcommitted
 TKGS
+bursty
+allotable
+appliveview
```

Comment:
All of these are not really "words" and should be fixed by wrapping code blocks in `<!-- cSpell:disable/enable -->` blocks

Reply:
bursty and allotable are.. appliveview.. is the name of a service..

From a pragmatic perspective, would we ever want cspell to alert if you say "appliveview"?

Reply:
I made all code snippets ignored